testbed/lib/main.dart

```diff
@@ -62,6 +67,26 @@ void main() {
   runApp(new MyApp());
 }
 
+Future<void> _windowMinMax() async {
```

Comment:
This method name isn't very clear on what it does.

Reply:
It was testing the channel communication for min/max value conversion. No longer required.